src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -167,50 +168,10 @@ void ResourceAllocationManagerImpl::ProcessApplicationPolicyUpdate() {
                         allowed_modules.end(),
                         std::back_inserter(disallowed_modules));
 
-    RCAppExtensionPtr rc_extention = GetApplicationExtention(app_ptr);
+    auto rc_extention = RCHelpers::GetRCExtension(**app);
     Resources::const_iterator module = disallowed_modules.begin();
     for (; disallowed_modules.end() != module; ++module) {
       ReleaseResource(*module, application_id);
-      if (rc_extention) {
-        rc_extention->UnsubscribeFromInteriorVehicleData(*module);
-      }
-      SendOnRCStatusNotification();
```

Comment:
@LuxoftAKutsan why we remove it?

Reply:
@LitvinenkoIra @yang1070 Now InteriorDataManager is responsible for subscriptions

Reply:
@LuxoftAKutsan I mean  _SendOnRCStatusNotification();_